src/components/Rockets/Rocket.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import Rockets from './rockets';
+import store from '../../redux/ConfigureStore';
+
+const MockMissionComponent = () => (
+  <Provider store={store}>
+    <Rockets />
+  </Provider>
+);
+const mockChildComponent = jest.fn();
+// eslint-disable-next-line react/display-name
```

Comment:
- [ ] Kindly remove all commented line code, It is always a good practice to keep your code clean from such comments. 😉 👍🏻